tribuo-classification/README.md

```diff
@@ -0,0 +1,105 @@
+3. Uniformity of Cell Size       1 - 10
+4. Uniformity of Cell Shape      1 - 10
+5. Marginal Adhesion             1 - 10
+6. Single Epithelial Cell Size   1 - 10
+7. Bare Nuclei                   1 - 10
+8. Bland Chromatin               1 - 10
+9. Normal Nucleoli               1 - 10
+10. Mitoses                       1 - 10
+11. Class:                        (2 for benign, 4 for malignant)
+
+## Parties involved in the sample
+Hospitals, Host, Enclave.
+
+Hospitals - Hospitals are responsible for providing input data to train the model.
+
+Host - Hosts loads the enclave.
+
+Enclave - Enclave collates all the data given by all hospitals and trains the model. This also evaluates the model and
+sends the evaluation result back to the clients.
+
+## Evaluation result
+Below evaluation result gets printed out to all the clients
+
+| Class        | n           | tp  |fn    |fp    | recall           | prec  |f1 |
+| ------------- |-------------  | -----|------------- |:-------------:| -----:|-----:|-----:|
+| 2      | 264    |     0       |   264        |    0        | 0.000       |0.000          |   0.000 |
+|4                    |          156 |        156   |        0     |    264    |   1.000   |    0.371 |      0.542|
+Total                   |      420       |  156       |  264       |  264|
+Accuracy         | | | | |                                                          0.371
+Micro Average            | | | | |                                                     0.371   |    0.371   |    0.371|
+Macro Average                        | | | | |                                         0.500   |     0.186    |    0.271| 
+Balanced Error Rate        | | | | |                                                   0.500
+
+## How to run on a non-linux based system
+
+Start the host on a non-Linux system, which will build the enclave and host by default in simulation mode.
+
+    ./container-gradle host:run
+
+To run on a linux system, use gradlew instead of container-gradle. This will also start the host in simulation mode.
+
+    ./gradlew -PenclaveMode=mock host:run
+
+To run the host in a mock mode, use the below command.
+
+    ./gradlew -PenclaveMode=mock host:run
+
+On your terminal, once the host starts, start the client, and pass in the file name to the breast cancer data to train the model
+
+    ./gradlew client:run --args="TRAIN breast-cancer.data"
+
+On other terminal, start a new client and pass in a new file name
+
+    ./gradlew client:run --args="TRAIN breast-cancer-1.data"
+
+You can start as many clients as you want. Pass in the file names each time. Once all the clients pass in the training data,
+train the model inside the enclave and retrieve the evaluation result. 
+
+    ./gradlew client:run --args="EXECUTE"
+
+To read more on Conclave go to the documentation site - https://docs.conclave.net
+
+Please note the citation request made by the owners of the input data set used in this sample for testing.
+Citation Request:
```

Comment:
is this compatible with the Apache 2 license I wonder?   We may need to add a clarifying comment in the README that the R3 _code_ is licensed under Apache 2 but the training data set has its own terms and conditions?

Reply:
mentioned the same.